astor/code_gen.py

```diff
@@ -50,22 +50,30 @@ def to_source(node, indent_with=' ' * 4, add_line_information=False,
                                 pretty_string)
     generator.visit(node)
     generator.result.append('\n')
-    return pretty_source(str(s) for s in generator.result)
+    if set(generator.result[0]) == set('\n'):
+        generator.result[0] = ''
+    return pretty_source(generator.result)
```

Comment:
We can remove turning everything into a string, now that we've fixed a bug farther down in the code that would leak `None` (and probably other things) into the result list.